src/debugAdapter/goDebug.ts

```diff
@@ -816,8 +825,17 @@ class GoDebugSession extends DebugSession {
 				}
 				const args = this.delve.isApiV1 ? <DebugVariable[]>outArgs : (<ListFunctionArgsOut>outArgs).Args;
 				verbose('functionArgs', args);
+				args.every(local => {
```

Comment:
I repeated this instead of creating a function because of the const and my understanding is that const variables can't be reassigned right? js noob here.

Reply:
Because of which `const`? The one on `args`?
The const only means that you cannot re-assign the array object. But it doesnt stop you from iterating over the contents and changing them.